remote-playback/watch-availability-promise-return-callback-id.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <title>
+    Test that the promise returns the callback id when calling
+    watchAvailability().
+  </title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="/common/media.js"></script>
+  <script>
+    async_test((t) => {
+      let v = document.createElement("video");
+      v.src = getVideoURI("/media/movie_5");
+
+      v.remote
+        .watchAvailability(() => {})
+        .then(
+          t.step_func_done((callbackId) => {
+            assert_true(typeof callbackId === "number");
+          }),
+          t.unreached_func()
+        );
+    }, "Test that the promise returns the callback id when calling watchAvailability().");
```

Comment:
"Test that the Promise returned by watchAvailability() resolves with a numeric callback id."